src/lib.rs

```diff
@@ -207,7 +207,7 @@ impl<T: Any + Send + Sync> DowncastSync for T {
 /// Adds downcasting support to traits that extend `downcast::Downcast` by defining forwarding
 /// methods to the corresponding implementations on `std::any::Any` in the standard library.
 ///
-/// See https://users.rust-lang.org/t/how-to-create-a-macro-to-impl-a-provided-type-parametrized-trait/5289
+/// See <https://users.rust-lang.org/t/how-to-create-a-macro-to-impl-a-provided-type-parametrized-trait/5289>
```

Comment:
Thanks!

Could you possibly regenerate `README.md` as well via `cargo readme >! README.md`?

Reply:
I keep forgetting :D

Reply:
Wait, I just realized :D This is a doc comment for the `impl_downcast` macro! Nothing to do for the readme haha